### PR TITLE
MICROVERSE: HTML Forms

### PR DESCRIPTION
**Pull request by anapdh**
The features created on this branch were:

A "Sign In" button, that when clicked directs to the login page;
The logo of the Company displayed right above the main content;
The main content as an input form to create an account for the website;
The form uses email, phone, and a password as the values of its inputs;
Clickable anchor links, but with no value.

### CODE REVIEW

**Comment by anapdh** [comment]
The features created on this branch were:

- A "Sign In" button, that when clicked directs to the login page;
- The logo of the Company displayed right above the main content;
- The main content as an input form to create an account for the website;
- The form uses email, phone, and a password as the values of its inputs;
- Clickable anchor links, but with no value.

File: index.html
Comment (teekaytech) [inline index.html]: The `Phone Number` field should be of `type="tel"`, and not `type="number"`. Kindly change the type of input.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__This is an awesome project :muscle:__
- All linter checks are green :green_circle: 
- Nice Readme :+1: 
- Proper use of HTML semantic tags :ok_hand: 

__However, there are some changes that are required to be fixed before this project can be approved. Please check my points below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

_This review will be based on the fact that you are expected to make the appearance of your page matches the mint's sign up page._



- [ ] Your pull request description should be exactly inside the description area, and should not be a comment. It should be the next thing immediately after your PR title.
- Kindly move your PR description to its right place. To do that, See the image below. Click on the `. . .` where the arrow is pointing to and select `Edit`. 



- [ ] The `fa-lock` icons inside the two buttons should be the same color (white)  as the text inside the buttons. Kindly fix this.

Source Page:


Your Page:


- [ ] Change the border color of your input boxes to match that of the original page.

- [ ] Observe closely the margins, white spaces, and padding of all the elements in the source page and try to use them to adjust your page. See the image below.

Source page:


Your page:




**Please make the required changes and submit another review request when you are done.**

* * *

**_You can reach out to me on twitter (@ola_lere), Github(@teekaytech) or slack (@taofeekolalere )_**
__Happy Coding :smiley:__

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## :wave: :wave: Hello!! Great effort!!

### [Mandatory] :red_circle: 

- [ ] MIT license does not correspond to the project. See comments

- [ ] Product logos does not match the original order.
image

- [ ] Project HTML is missing ```<body>``` tag. See comments

- [ ] ```<input>``` should be outside the ```<label>``` tag. example:
```
  ...
  <label for="male">Male</label>
  <input type="radio" name="gender" id="male" value="male"><br>
  <label for="male">Male</label>
  <input type="radio" name="gender" id="male" value="male"><br>
  ...
```

### [OPTIONAL] :large_blue_circle: 

- [ ] For future projects choose a meaningful branch name. Example:
**Actual**: ```form```
**Improved** :+1: : ```mint-signup-form```
This helps readability and helps the reader and the programmer understand the intent of the branch.

### [Comments] :green_circle: 

Is Great that you added a license to your project. Is a great habit and best practice. Please follow the next steps to create your own version of the MIT license.
- [ ] Create a copy of the license you're using and add it to the project repository in a ```license.md``` file
- [ ] Update it with the name(s) of the owners of the project.
image
- [ ] Update the License link in the README 

The ```<body>``` tag is an important part of the HTML file structure; is the one that defines the document's body, and contains all the contents of an HTML document, such as headings, paragraphs, images, hyperlinks, tables, lists, etc. 

If you notice, the page still renders as intended, this is because the browsers are kind of smart and try to interpret the code as best as possible. This will not always be the case, so its important to maintain the HTML structure as it has to be. 

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Review by teekaytech — APPROVED** [review]
# Status: Approved :green_circle: 

### Congratulations team, this project has passed all the mandatory requirements. It is therefore approved :heavy_check_mark:

You did an amazing job in fixing all the changes required by the last reviewer, keep up the good work :muscle: 
Good luck with your next project.

**I'm @taofeekolalere on slack.
Happy Coding :man_technologist:**

**Pull request merged**
